server/src/routes/api/v1/trailsRouter.js

```diff
@@ -28,4 +31,25 @@ trailsRouter.get("/:id", async (req, res) => {
   }
 })
 
+trailsRouter.post("/", async (req, res) => {
+  const { body } = req
+  const formInput = cleanUserInput(body)
+  const { trailName, trailLength, trailLocation, trailDescription, estimateTime } = formInput
+
+  try {
+    const newTrail = await Trail.query().insertAndFetch({
+      trailName,
+      trailLength,
+      trailLocation,
+      trailDescription,
+      estimateTime,
+    })
+    return res.status(201).json({ trail: newTrail })
+  } catch (error) {
+    if (error instanceof ValidationError) {
+      return res.status(422).json({ errors: error.data })
+    }
```

Comment:
We also need a fallback `res.status(500)`

Reply:
res.status(500) added!